app/Application.php

```diff
@@ -9,36 +9,42 @@
 
 class Application
 {
+    protected $messages;
     protected $historyRepository;
 
     public function __construct()
     {
+        $DS = DIRECTORY_SEPARATOR;
+
+        $this->messages = file_get_contents("locale{$DS}en.json");
+        $this->messages = json_decode($this->messages, true);
+
         $this->historyRepository = new HistoryRepository();
     }
 
     public function run()
     {
         $isRunning = true;
-        info_box('', 'Welcome to the calculator app!', '', 'Print "help" to learn more about the app.', '');
+        info_box('', "{$this->messages['textWelcome1']}", '', "{$this->messages['textWelcome2']}", '');
```

Comment:
please remove useless double quotes and brackets in all other cases too

```suggestion
        info_box('', $this->messages['textWelcome1'], '', $this->messages['textWelcome2'], '');
```